rs/tests/ckbtc/BUILD.bazel

```diff
@@ -56,12 +55,33 @@ rust_library(
     deps = DEPENDENCIES,
 )
 
+oci_tar(
+    name = "minica.tar",
+    image = "@minica",
+    repo_tags = ["minica:image"],
+)
+
+uvm_config_image(
+    name = "ckbtc_uvm_config_image",
+    srcs = [
+        ":minica.tar",
```

Comment:
What about moving the equivalent [`//rs/tests/networking/canister_http:minica.tar`](https://sourcegraph.com/github.com/dfinity/ic/-/blob/rs/tests/networking/canister_http/BUILD.bazel?L28-32) to [`/rs/tests/BUILD.bazel`](https://sourcegraph.com/github.com/dfinity/ic/-/blob/rs/tests/BUILD.bazel) and removing this one?